addon/components/frost-checkbox.js

```diff
@@ -5,10 +5,24 @@ import _ from 'lodash/lodash'
 export default Ember.Component.extend({
   layout: layout,
   classNames: ['frost-checkbox'],
+  classNameBindings: ['size'],
 
   didInitAttrs () {
     const checked = this.attrs ? this.attrs.checked : false
     this.set('isChecked', checked)
+
+    const checkboxSize = this.get('size')
+    switch (checkboxSize) {
```

Comment:
No necessary, but I did it this way to catch the case when no size is specified.